modules/plugins/wfly-10/src/main/java/org/rhq/modules/plugins/wildfly10/ModuleOptionsComponent.java

```diff
@@ -572,34 +572,45 @@ public void updateResourceConfiguration(ConfigurationUpdateReport report) {
 
                 //locate specific node and populate the config
                 String loginModuleIdentifier = attribute + ":";
+               
                 int index = path.indexOf(loginModuleIdentifier);
                 String loginModuleIndex = path.substring(index + loginModuleIdentifier.length());
                 String[] split = loginModuleIndex.split(",");
-                int lmi = Integer.valueOf(split[0]);//Ex 0,1,30
-                Value valueObject = currentAttributeState.get(lmi);
-
-                //retrieve current config changes    
-                Configuration conf = report.getConfiguration();
-                //list current conf stated being defined.
-                String mapKey = (String) conf.getMap().keySet().toArray()[0];
-                PropertyMap mapType = (PropertyMap) conf.getMap().get(mapKey);
-                //insert update logic.
-                //set this new state passed in as state of the attribute.
-                LinkedHashMap<String, Object> currentOptions = new LinkedHashMap<String, Object>();
-                for (String propertyKey : mapType.getMap().keySet()) {
-                    currentOptions.put(propertyKey, ((PropertySimple) mapType.get(propertyKey)).getStringValue());
+                
+                Value valueObject=null;
```

Comment:
This type of formatting is not consistent with the rest of the code. Try to add some more whitespace like the previous code has, such as ``try {`` and ``} catch (..) {`` as well as ``valueObject != null``.